### DISCUSSION TITLE
Failed to import tailwind in React

### DISCUSSION THREAD

**Discussion by dgwyer** [tailwindlabs/tailwindcss#2774] [Help]
I just span up a new `create-react-app` project and in the `App.js` file added:

`const tailwind = require('tailwindcss');`

But this is causing a compilation error:

`TypeError: The "original" argument must be of type Function`

I want to try and compile Tailwind CSS dynamically on demand as discussed in the thread linked below but I can't get past the import error above. Is there a new way to include the Tailwind function?

https://github.com/tailwindlabs/tailwindcss/issues/977

**Comment by dgwyer** [comment]
I couldn't get this working locally so I tried it on code sandbox and it worked:
https://codesandbox.io/s/quizzical-einstein-27bn1?file=/src/App.js

However, if I export the exact same project from code sandbox locally then it fails to compile. I get the error "TypeError: The "original" argument must be of type Function". I'm confused as to why the same project won't work locally.

![image](https://user-images.githubusercontent.com/1482075/99421757-a2016b80-28f6-11eb-96bd-0d359c4c16e8.png)

**Comment by joemsak** [comment]
I just created a brand new react app today, followed the Tailwind documentation and I get the same error trying to run the local development server in Node 14.16.1

Here is my source code: https://github.com/joemsak/joemsak.github.io

EDIT: My bad, I left in some bad "trial and error" code in my App.js file that interfered and gave the same error in this discussion
